container/rkt/handler.go

```diff
@@ -99,20 +99,19 @@ func newRktContainerHandler(name string, rktClient rktapi.PublicAPIClient, rktPa
 	})
 	if err != nil {
 		return nil, err
-	} else {
-		var annotations []*rktapi.KeyValue
-		if parsed.Container == "" {
-			pid = int(resp.Pod.Pid)
-			apiPod = resp.Pod
-			annotations = resp.Pod.Annotations
+	}
+	annotations := resp.Pod.Annotations
+	if parsed.Container != "" {
+		if contAnnotations, ok := findAnnotations(resp.Pod.Apps, parsed.Container); !ok {
+			glog.Warningf("couldn't find app %v in pod", parsed.Container)
 		} else {
-			var ok bool
-			if annotations, ok = findAnnotations(resp.Pod.Apps, parsed.Container); !ok {
-				glog.Warningf("couldn't find application in Pod matching %v", parsed.Container)
-			}
+			annotations = append(annotations, contAnnotations...)
 		}
-		labels = createLabels(annotations)
+	} else {
```

Comment:
was having a hard time lining up the code from the diff, I see it now.

but yes, it's the pod container